orttraining/orttraining/training_ops/cpu/controlflow/wait.cc

```diff
@@ -16,7 +16,7 @@ void wait_event_in_tensor(const Tensor& event_id_tensor) {
     OrtEventPool::GetInstance().WaitEvent(event_id);
     // BUGBUG: seems this would cause hang when a event is being waited more than once
     // Destory the recorded event.
```

Comment:
Is this comment still valid?

Reply:
No. Will remove it.

---
In reply to: [476890263](https://github.com/microsoft/onnxruntime/pull/4916#discussion_r476890263) [](ancestors = 476890263)